src/d3a/models/strategy/external_strategies/load.py

```diff
@@ -34,16 +34,24 @@ class LoadExternalMixin(ExternalMixin):
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
 
-    def event_activate(self):
-        super().event_activate()
-        self.redis.sub_to_multiple_channels({
+    @property
+    def _channel_dict(self):
+        return {
             f'{self.channel_prefix}/register_participant': self._register,
             f'{self.channel_prefix}/unregister_participant': self._unregister,
             f'{self.channel_prefix}/bid': self._bid,
             f'{self.channel_prefix}/delete_bid': self._delete_bid,
             f'{self.channel_prefix}/list_bids': self._list_bids,
             f'{self.channel_prefix}/device_info': self._device_info,
-        })
+            }
+
+    @property
+    def channel_dict(self):
```

Comment:
Any specific reason to wrap same private property to public?

Reply:
Yes, in order to overwrite the `self._channel_dict` in the LoadForecastExternalStrategy.

Reply:
Why not re-signature the existing `_channel_dict ` to `channel_dict ` ?

Reply:
Because i need both. Please have a look at the LoadForecastExternalStrategy.
 I can not update a property (`channel_dict`), but i can change the 
`_channel_dict` which is not a property.

Reply:
umm, both are property.

Reply:
oh ,yes, sorry, i had so many forth and backs here that i lost track. 
But here too it is a matter of taste. I either do as I did or I have to overwrite the event_activate method in the LoadForecastExternalStrategy. 
Maybe @spyrostz  is best to be ask here for the best practice.

Reply:
I would actually remove the private property and keep only the public property, unless I am not seeing something.

Reply:
ok guys, i gave it yet another look. I dont want to subscribe to `f'{self.channel_prefix}/set_power_forecast'` channel for the normal `LoadExternalMixin`. So i have to change the list of channels in the `LoadForecastExternalStrategy`. I dont see another way of doing it the way i did. I can not overwrite the `event_activate` because i need to call the event_activate of the LoadHours Strategy there.
But maybe there is another way, could you please tell me? Thanks

Reply:
Updating channel dict inside the constructor of this new class could be an option like `self.channel_dict.update({f'{self.channel_prefix}/set_load_power_forecast': self._set_power_forecast})`

Reply:
No, this wont work, as it is a property. You can not overwrite a property.

Reply:
This property returns dictionary to which you are updating, I haven't tried yet but that was from the top of my head to give a try

Reply:
I have tried it in the past. 
yes, indeed, the property returns a dict, but it is really a function. You can not call `.update()` on a function. 
(already pycharm is complaining about it. )

Reply:
what i could do here is to initiate a dictionary `channel_dict` in the constructor and follow the approach you suggested. (so removing the property function.

Reply:
You can also use `super()`